components/tabs/tabs.scss

```diff
@@ -0,0 +1,96 @@
+.tabs {
+  margin-bottom: 1rem;
+}
+
+.tabs__title {
+  .no-js & { // We visually hide the title with u-vh and need to undo with no-js
+    position: relative;
+    width: auto;
+    height: auto;
+    margin: 0 0 1rem;
+  }
+}
+
+// Tabs - list
+.tabs__list {
+  margin: 0;
```

Comment:
```suggestion
  margin: 0;
  padding-left: 0;
```

Reply:
I've kept a small amount of padding on the left as it didn't look quite right when the first tab was not selected. Let me know what you think.

Reply:
@RhodGriffith review the PR when you get a minute as this was from your prototypes. Ta.

Reply:
@boxadesign I think it looks better without any padding on the left, can we remove this? 
Padding should be 0.5rem between tabs (which is the orange area in the screenshot). I think it's inheriting additional padding from somewhere, can we close that space to make it a bit more compact?
<img width="606" alt="screen shot 2018-11-06 at 12 57 00" src="https://user-images.githubusercontent.com/28897806/48066056-6e968d00-e1c4-11e8-834b-05b963cd1c37.png">

Reply:
Yep fair enough. Will push up a change in a mo